src/libm/mkrename.c

```diff
@@ -61,10 +68,10 @@ int main(int argc, char **argv) {
   else {
     char *wdp = argv[1];
     char *wsp = argv[2];
-    char *vdoublename = argv[3];
-    char *vfloatname = argv[4];
-    char *vintname = argv[5];
-    char *vint2name = argv[6];
+    char *vdoublename = argv[3], *vdoublename_escspace = escapeSpace(vdoublename);
```

Comment:
Can you do it the other way around, so that you don't have to change the variable used in the printf calls? We have a patch downstream that adds the AArch64 Vector PCS attribute [1] to the renaming functions (of course, we will upstream it), your changes will you be easier to merge if you leave the original name in the printf calls.

[1] https://developer.arm.com/products/software-development-tools/hpc/arm-compiler-for-hpc/vector-function-abi

Reply:
Not all the arguments of printf calls are space-escaped.
I don't think of a good way of fulfilling your request.

Reply:
In altivec or VSX intrinsics, vector data types are like "vector float".

https://gcc.gnu.org/onlinedocs/gcc/PowerPC-AltiVec_002fVSX-Built-in-Functions.html
http://openpowerfoundation.org/wp-content/uploads/resources/leabi/content/dbdoclet.50655245_56497.html

Reply:
h
Hi @shibatch , sorry for bothering you again with this. I had a second (or maybe third!!) thought, and if my understanding is correctly, the following variables don't have space in their names, unless when targeting VSX:
```
vdoublename
vfloatname
vintname
vint2name
```
 
Given that the function `escapeSpace` do not modify the values of those variables when not targeting VSX, I'd rather have you redefine them as follows:

```
char *vdoublename = escapeSpace(argv[3]);
char *vfloatname = escapeSpace(argv[4]);
char *vintname = escapeSpace(argv[5]);
char *vint2name = escapeSpace(argv[6]);
```

This should keep the functionality of the current patch, without the need to modify the variables name in the printf calls.